### PR TITLE
Task/frontend64 implement tooltips

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
[Implement tooltips](https://trello.com/c/LC8EGtUW/64-implement-tooltips)

File: frontend/package.json [hunk 1]
```diff
@@ -24,9 +24,9 @@
     "query-string": "7.1.1",
     "react": "18.2.0",
     "react-avatar-editor": "13.0.0",
+    "react-content-loader": "6.2.0",
```
Comment (droztoff) [inline frontend/package.json]: why you cant use "react-loading-skeleton" instead this dependecy?

File: frontend/src/components/common/header/header.tsx [hunk 2]
```diff
@@ -76,6 +78,12 @@ const Header: FC<HeaderProps> = ({
 
   return (
     <header className={styles['header']}>
+      <ReactTooltip
+        place="bottom"
+        backgroundColor={isLightTheme ? '#c3cfc0' : '#000000'}
```
Comment (droztoff) [inline frontend/src/components/common/header/header.tsx]: this logic can be put into an object and calculations can be performed there (as an example videos-block.tsx)

File: frontend/src/components/common/sidebar/sidebar.tsx [hunk 3]
```diff
@@ -68,10 +72,18 @@ const Sidebar: FC<SidebarProps> = ({ configRoutePages, activeRouteId, isSidebarO
...
+                      <li data-tip={page.textLink}>
                         <Icon name={page.iconName} width="24" height="24" />
                         <span className={styles['link-name']}>{page.textLink}</span>
                       </li>
+                      {!isSideBarOpen && (
+                        <ReactTooltip
+                          place="right"
+                          backgroundColor={isLightTheme ? '#c3cfc0' : '#000000'}
+                          effect="solid"
+                          textColor={isLightTheme ? '#000000' : '#ffffff'}
```
Comment (droztoff) [inline frontend/src/components/common/sidebar/sidebar.tsx]: same take out the logic

File: frontend/package.json (hunk 1 above)
Comment (ghost) [reply frontend/package.json]: i use this dependency in previos task to implement skeleton to video card from search page in my opinion it is easier to use

File: frontend/src/components/common/header/header.tsx (hunk 2 above)
Comment (ghost) [reply frontend/src/components/common/header/header.tsx]: done

File: frontend/src/components/common/sidebar/sidebar.tsx (hunk 3 above)
Comment (ghost) [reply frontend/src/components/common/sidebar/sidebar.tsx]: done

**Review by droztoff — APPROVED** [review]

**Pull request merged**